### PR TITLE
fix(webapp): implement i18n

### PR DESCRIPTION
**Pull request by tetogomez**
### What does this PR do?

-  implement i18n

### Steps to test

1. first step
2. next step

#### CheckList

- [] Follow proper Markdown format
- [] The content is adequate
- [] The content is available in both english and spanish
- [] I Ran a spell check

### CODE REVIEW

File: public/locales/en/common.json [hunk 1]
```diff
@@ -1,6 +1,57 @@
...
+  "routes": {
+    "/": "Home",
+    "voters": "Voters",
+    "vote": "Vote",
+    "about": "About",
+    "mustLogin": "You must login",
+    "successfulVote": "Successful vote",
+    "somethingWrong": "Something went wrong, try again"
+  },
+  "about": {
```
Comment (AngeloCG97) [inline public/locales/en/common.json:16]: On this file the tag say about but in spanish file say sobre, this is correct?

File: src/components/Layout/Header/index.tsx [hunk 2]
```diff
@@ -83,6 +86,12 @@ const Header: React.FC<HeaderProps> = ({ onDrawerToggle }) => {
     [message]
   )
 
+  useEffect(() => {
+    setPathName(asPath.replace('/', ''))
+  }, [asPath, setPathName])
+
+  console.log(pathName)
```
Comment (AngeloCG97) [inline src/components/Layout/Header/index.tsx:93]: Please remove this console

**Review by AngeloCG97 — CHANGES_REQUESTED** [review]

File: public/locales/en/common.json [hunk 3]
```diff
@@ -1,6 +1,57 @@
...
-  "home": "Home",
   "contact": "Contact",
   "copyright": "Copyright",
-  "support": "Support"
+  "support": "Support",
+  "es": "Spanish",
+  "en": "English",
+  "routes": {
+    "/": "Home",
+    "voters": "Voters",
```
Comment (xavier506) [inline public/locales/en/common.json:9]: We should call Voters Members or Eden Members , vote and voters seems a beet confusing to me #64

File: public/locales/en/common.json [hunk 4]
```diff
@@ -1,6 +1,57 @@
...
+    "vote": "Vote",
+    "about": "About",
+    "mustLogin": "You must login",
+    "successfulVote": "Successful vote",
```
Comment (xavier506) [inline public/locales/en/common.json:13]: Voted Successfully

File: public/locales/en/common.json [hunk 5]
```diff
@@ -1,6 +1,57 @@
...
+    "somethingWrong": "Something went wrong, try again"
+  },
+  "about": {
+    "votersMetaTitle": "About",
+    "whatIsEden": "What is Eden Proxy?",
+    "edenDescription": "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse maximus molestie laoreet. Vestibulum in varius neque. Praesent at finibus nisi. Vestibulum sagittis nec ligula non vulputate. Donec eu nulla rutrum, volutpat eros sit amet, tempor augue. Maecenas risus quam, efficitur nec enim quis, pretium semper turpis. Duis id arcu fringilla, vulputate nulla eget, lobortis nisi. Cras quis feugiat lectus. Donec vestibulum et dolor a vestibulum. In interdum, ex nec porttitor consequat, nulla tellus malesuada velit, eu pretium augue dolor ut nisi. Proin pretium convallis urna, quis dictum nulla ornare eu. Nulla et mollis lorem. Nulla pretium lacus tortor, sit amet dignissim mi tincidunt non. Suspendisse ut maximus dui. Ut massa mi, luctus a magna ut, porta viverra libero.",
```
Comment (xavier506) [inline public/locales/en/common.json:19]: can we replace this placeholder text with some real text?

File: public/locales/en/common.json [hunk 6]
```diff
@@ -1,6 +1,57 @@
...
+    "question3": "3. No more than one BP with the same ownership.",
+    "question4": "4. Acknowledge that BPs can be unlisted at any time if engage with any ativities above.",
+    "disclaimer": "Disclaimer:",
+    "point1": "As a grace-period, BPs are not going to be requested to drop their existing vote-buy until they receive 30m to stay in the pay range.",
+    "point2": "There's is no subjectivity in the whitelist other than above 4 criterias.",
+    "point3": "As a grace-period, BPs are not going to be requested to drop their existing vote-buy until they receive 30m to stay in the pay range.",
+    "shouldVote": "Why should I vote?",
+    "shouldVoteResponse": " Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse maximus molestie laoreet. Vestibulum in varius neque. Praesent at finibus nisi. Vestibulum sagittis nec ligula non vulputate. Donec eu nulla rutrum, volutpat eros sit amet, tempor augue. Maecenas risus quam, efficitur nec enim quis, pretium semper turpis. Duis id arcu fringilla, vulputate nulla eget, lobortis nisi. Cras quis feugiat lectus. Donec vestibulum et dolor a vestibulum."
```
Comment (xavier506) [inline public/locales/en/common.json:31]: do we have this text somewhere?

File: public/locales/en/common.json [hunk 7]
```diff
@@ -1,6 +1,57 @@
...
+  "vote": {
+    "voteMetaTitle": "Vote",
+    "mustLogin": "You must login",
+    "successfulVote": "Successful vote",
+    "somethingWrong": "Something went wrong, try again",
+    "loadMore": "Load More",
+    "voteSelected": "Vote Selected"
+  },
+  "voters": {
+    "votersMetaTitle": "Voters",
```
Comment (xavier506) [inline public/locales/en/common.json:42]: Eden Members

File: public/locales/es/common.json [hunk 8]
```diff
@@ -1,6 +1,57 @@
...
   "contact": "Contacto",
-  "copyright": "Copyright",
-  "support": "Soporte"
+  "copyright": "Derechos de autor",
+  "support": "Soporte",
+  "es": "Español",
+  "en": "Inglés",
+  "routes": {
+    "/": "Hogar",
+    "voters": "Votantes",
```
Comment (xavier506) [inline public/locales/es/common.json:9]: Miembros

File: public/locales/en/common.json (hunk 5 above)
Comment (tetogomez) [reply public/locales/en/common.json:19]: I think Hahn is working on it!

**Comment by AngeloCG97** [comment]
LGTM

**Review by AngeloCG97 — APPROVED** [review]
LGTM

**Pull request merged**